tag/feature.py

```diff
@@ -164,7 +184,10 @@ def __ge__(self, other):
     def __iter__(self):
         """Generator iterates through a feature and all its subfeatures."""
         sorted_features = list()
-        self._visit(L=sorted_features, marked={}, tempmarked={})
+        root = self
+        if self.is_pseudo:
+            root = self.children[0].multi_rep
```

Comment:
First child (`children[0]`) *should* be the multi rep anyway. Maybe I should enforce this with an assertion rather than re-referencing.